package.json

```
@@ -1,14 +1,14 @@
 {
   "name": "tsprogen",
-  "version": "1.2.0",
+  "version": "1.2.1",
   "description": "A CLI project generator for setting up TypeScript + Node projects.",
   "main": "./prod/bin.js",
   "bin": {
     "tsprogen": "./bin/tsprogen"
   },
   "scripts": {
     "build": "tsc",
```

```suggestion
  "scripts": {
    "prebuild": "rm -rf ./prod",
    "build": "tsc",
```

While we are in here, let's add a prebuild script to remove the existing `prod` directory - this'll prevent conflicts when building and publishing.